src/vs/workbench/contrib/terminal/browser/terminalEditorService.ts

```diff
@@ -55,6 +58,46 @@ export class TerminalEditorService extends Disposable implements ITerminalEditor
 		}));
 	}
 
+	private _getActiveTerminalEditors(): IEditorInput[] {
+		return this._editorService.visibleEditors.filter(e => e instanceof TerminalEditorInput && e.terminalInstance?.instanceId);
+	}
+
+	private _getActiveTerminalEditor(): TerminalEditor | undefined {
+		return this._editorService.activeEditorPane instanceof TerminalEditor ? this._editorService.activeEditorPane : undefined;
+	}
+
+	findPrevious(): void {
+		const editor = this._getActiveTerminalEditor();
+		editor?.showFindWidget();
+		editor?.getFindWidget().find(true);
+	}
+
+	findNext(): void {
+		const editor = this._getActiveTerminalEditor();
+		editor?.showFindWidget();
+		editor?.getFindWidget().find(false);
+	}
+
+	getFindState(): FindReplaceState {
+		const editor = this._getActiveTerminalEditor();
+		return editor?.findState!;
```

Comment:
Don't think the `!` is needed

```suggestion
		this._getActiveTerminalEditor()?.findState;
```